src/telemetry/registerListeners.ts

```diff
@@ -5,25 +5,32 @@
 
 import { TextDocument, workspace } from 'vscode';
 import { IActionContext, registerEvent } from 'vscode-azureextensionui';
-import { ext } from './extensionVariables';
-
-let lastUploadTime: number = 0;
-const hourInMilliseconds = 1000 * 60 * 60;
+import { ext } from '../extensionVariables';
 
 export function registerListeners(): void {
     if (ext.telemetryOptIn) {
         registerEvent('dockerfilesave', workspace.onDidSaveTextDocument, async (context: IActionContext, doc: TextDocument) => {
-            // If it's not a Dockerfile, or last upload time is within an hour, skip

```

Comment:
So why remove this logic now? Or why did we have it here in the first place?

Reply:
In the past it was done in the name of performance. I had a couple reasons for removing it now:
- VSCode itself is sending (multiple) telemetry events every time a file is saved, with no rate limiting
- For the purposes of experimentation, it's better to send the event every time